### PR TITLE
updated healthcheck to 60m

### PR DESCRIPTION
**Pull request by Manali-Jain-squareops**
### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: config/0chain_blobber.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ challenge_response:
   max_retries: 20
 
 healthcheck:
-  frequency: 60s # send healthcheck to miners every 60 seconds
+  frequency: 60m # send healthcheck to miners every 60 seconds
```
Comment (boddumanohar) [inline config/0chain_blobber.yaml]: please also change the comment. 

`# send healthcheck to miners every 1 hour`

**Review by boddumanohar — APPROVED** [review]
LGTM

File: config/0chain_blobber.yaml (hunk 1 above)
Comment (Manali-Jain-squareops) [reply config/0chain_blobber.yaml]: updated

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**